modules/getting-started/con-techdocs-configure-storage.adoc

```diff
@@ -0,0 +1,5 @@
+[id="con-techdocs-configure-storage_{context}"]
+
+= Configuring storage for TechDocs files
```

Comment:
```suggestion
// Module included in the following assemblies:
//
// * assemblies/assembly-techdocs-plugin.adoc

:_mod-docs-content-type: CONCEPT
[id="con-techdocs-configure-storage_{context}"]
= Configuring storage for TechDocs files
```

Reply:
Is this really necessary as its own module? If it's only being included in this one place, I would just make it a H2  section in the assembly, or include this line in the intro content about TechDocs.